tests/e2e/README.md

```diff
@@ -178,6 +185,24 @@ Please see golang testing options for more information.
   make e2e_simple E2E_ARGS="--skip_setup --skip_cleanup --namespace=e2e -istioctl ~/istioctl-osx --auth_enable --test.run TestSimpleIngress"
   ```
 
+* Running pilot e2e tests on minikube with custom hubs/tags:
+
+```bash
+go test -v ./tests/e2e/tests/pilot \
+  -mixer_tag ${MYTAG} \
```

Comment:
In separate PR - can you remove this, I believe we have a single hub/tag that overrides all. 

In fact - HUB and TAG env variables should be used so "go test -v with no args should work !